packages/sample-app/sample_app/openai_guardrails_example.py

```
@@ -0,0 +1,183 @@
+OpenLLMetry tracing. The guardrails library wraps the OpenAI client to
+provide automatic input/output validation and safety checks.
+
+The trace context is properly maintained across the wrapper's internal calls,
+ensuring complete observability of the request flow.
+"""
+
+import os
+import json
+import asyncio
+from pathlib import Path
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
```

Flake8 (7.3.0)

[error] 'opentelemetry.sdk.trace.export.ConsoleSpanExporter' imported but unused

(F401)